### PR TITLE
[nomerge] remove unused import.

### PR DESCRIPTION
**Pull request by takayahilton**

### CODE REVIEW

**Review by hrhino — APPROVED** [review]
Nice, thanks!

**Comment by SethTisue** [comment]
This is outstanding, thank you!

I think it might be hard to merge it onto the 2.13.x branch. Can I interest you in marking this one `[nomerge]` and also submitting a separate 2.13 PR?

Is there a possibility of guarding against regressions in this area by enabling `-Ywarn-unused:imports` in our build?

**Comment by takayahilton** [comment]
@SethTisue 
ok.
I will submit a 2.13 PR.
And also add scalac option to `-Ywarn-unused: imports`.

**Comment by som-snytt** [comment]
On 2.13, https://github.com/scala/scala/pull/6964 but I think it isn't permanent yet. I see a comment about waiting for re-starr.

The similar PR was https://github.com/scala/scala/pull/6806 

Note that it's `-Xlint` on the 2.13 PR.

**Review by diesalbla — APPROVED** [review]

**Comment by dwijnand** [comment]
Needs a rebase now as src/compiler/scala/tools/nsc/backend/jvm/opt/BytecodeUtils.scala conflicts.

File: build.sbt [hunk 1]
```diff
@@ -817,6 +815,7 @@ lazy val test = project
     fork in IntegrationTest := true,
     // enable this in 2.13, when tests pass
     //scalacOptions in Compile += "-Yvalidate-pos:parser,typer",
+    scalacOptions -= "-Ywarn-unused:imports",
```
Comment (dwijnand) [inline build.sbt:818]: I'm fine with doing this, but how come you did it?

**Review by dwijnand — APPROVED** [review]

File: build.sbt (hunk 1 above)
Comment (takayahilton) [reply build.sbt:818]: I don't know how reason, but `partest` fails when adding `-Ywarn-unused: imports`.

Comment (som-snytt) [reply build.sbt:818]: The 2.13 PR does something similar. Test compilations receive these args, too, with fatal warnings.

**Comment by som-snytt** [comment]
BTW, I'm preparing a quick update to 2.13 since the last clean-up PR.

**Comment by som-snytt** [comment]
Thanks again for setting the flag. It was so nice seeing the extra help in my sbt window today. I was "optimizing imports" as they say in IDE lingo.

**Pull request merged**